### PR TITLE
Change the Roslyn solution analyzers to Portable

### PR DESCRIPTION
**Pull request by agocke**
Self-explanatory

@tmat @jaredpar

### CODE REVIEW

File: src/Diagnostics/Roslyn/Core/RoslynDiagnosticAnalyzers.csproj [hunk 1]
```diff
@@ -105,6 +106,9 @@
       <LastGenOutput>RoslynDiagnosticsResources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
```
Comment (jaredpar) [inline src/Diagnostics/Roslyn/Core/RoslynDiagnosticAnalyzers.csproj]: Is this necessary?

**Comment by jaredpar** [comment]
:+1:

File: src/Diagnostics/Roslyn/Core/RoslynDiagnosticAnalyzers.csproj (hunk 1 above)
Comment (agocke) [inline src/Diagnostics/Roslyn/Core/RoslynDiagnosticAnalyzers.csproj]: Ugh, I don't know how that got back in there -- I thought I removed it.

**Comment by agocke** [comment]
retest this please

**Pull request merged**